### PR TITLE
Add AnimateAnyone tutorial

### PR DESCRIPTION
**Pull request by itrushkin**
CVS-131132

- [x] Gradio interface
- [x] README

This tutorial required to change the model code for conversion, thus it references to the forked repo of the original implementation

### CODE REVIEW

File: notebooks/animate-anyone/animate-anyone.ipynb [hunk 1]
```diff
@@ -0,0 +1,1349 @@
+{
```
Comment (aleksandr-mokrov) [inline notebooks/animate-anyone/animate-anyone.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.        ov_image_encoder = nncf.compress_weights(ov_image_encoder)
</pre><p>nncf is not defined</p><p>Add some description and link on NNCF Compression Weights documentation page</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1846/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (aleksandr-mokrov) [inline notebooks/animate-anyone/animate-anyone.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #54.            gr.Slider(label="Number of inference steps", value=30, minimum=15, maximum=100)
</pre><p>Could you add video length</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1846/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

File: notebooks/animate-anyone/animate-anyone.ipynb
Comment (aleksandr-mokrov) [inline notebooks/animate-anyone/animate-anyone.ipynb:1]: Replace it by link: https://github.com/openvinotoolkit/openvino_notebooks/assets/76171391/1d0a8552-a296-4bf4-8717-19ef095b4c2c

File: notebooks/animate-anyone/animate-anyone.ipynb (hunk 1 above)
Comment (itrushkin) [reply notebooks/animate-anyone/animate-anyone.ipynb:1]: <p>Model was converted with fixed height, width and video length to reduce memory consumption during inference.</p>

Comment (itrushkin) [reply notebooks/animate-anyone/animate-anyone.ipynb:1]: <p>Done</p>

File: notebooks/animate-anyone/animate-anyone.ipynb
Comment (itrushkin) [reply notebooks/animate-anyone/animate-anyone.ipynb:1]: Done

File: notebooks/animate-anyone/animate-anyone.ipynb (hunk 1 above)
Comment (eaidova) [reply notebooks/animate-anyone/animate-anyone.ipynb:1]: <p>still not see any relevant explanations...</p>

File: notebooks/animate-anyone/animate-anyone.ipynb [hunk 2]
```diff
@@ -0,0 +1,1652 @@
...
```
Comment (eaidova) [inline notebooks/animate-anyone/animate-anyone.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    os.makedirs(local_dir, exist_ok=True)
</pre><p>why do you mix os.path and Path objects, all used in this cell path-oriented commands can be performed with Path object without using additional libs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1846/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (eaidova) [inline notebooks/animate-anyone/animate-anyone.ipynb:1]: <p>please add info about nncf and weighs compression usage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1846/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

**Review by eaidova — APPROVED** [review]

**Pull request merged**